webpack/scenes/ContentViews/Details/Filters/ContentViewFilterDetailsHeader.js

```diff
@@ -11,10 +11,16 @@ const ContentViewFilterDetailsHeader = ({
   cvId, filterId, details, setShowAffectedRepos,
 }) => {
   const {
-    type, name, inclusion, description,
+    type, name, inclusion, description, rules,
   } = details;
+  let errataByDate = false;
+  if (type === 'erratum' && rules[0]?.types) errataByDate = true;
   const repositoryType = repoType(type);
-  const displayedType = type ? capitalize(type.replace(/_/g, ' ')) : '';
+  const displayedType = () => {
+    if (errataByDate) return __('Errata - by date range');
+    if (type) return capitalize(type.replace(/_/g, ' '));
```

Comment:
We should eliminate all cases of this custom utility helper. 
https://lodash.com/docs/4.17.15#capitalize

Reply:
Updated the PR with lodash..Thanks for pointing to it.. 👍🏽